bigframes/dataframe.py

```diff
@@ -1195,7 +1195,7 @@ def to_pandas(
                 The seed for the uniform downsampling algorithm. If provided, the uniform method may
                 take longer to execute and require more computation. If set to a value other than
                 None, this will supersede the global config.
-            ordered (bool, default True):
+            ordered (bool, default None):
```

Comment:
Let's say what `None` does. (I assume it respects any `sort_values` calls but may be non-deterministic in the case of not a total orderign?)

Reply:
`None` just uses the session default, there is still only 2 modes (True/False). Extended arg description in new revision